src/components/Standings/Standings.js

```diff
@@ -5,85 +5,125 @@ import {
   View,
   StyleSheet,
   TouchableHighlight,
-  ActivityIndicator
+  ActivityIndicator,
+  Switch
 } from 'react-native'
 import Place from './place'
 
 import { withFirebaseHOC } from '../../../config/Firebase'
 
-const styles = StyleSheet.create({
-  container: {
-    width: '98%',
-    marginBottom: 5,
-    borderColor: '#FFC72C',
-    borderWidth: 1,
-    borderRadius: 15,
-    overflow: 'hidden'
-  },
-  ListView: {
-    paddingLeft: 5,
-    marginTop: 5,
-    paddingTop: 5,
-    paddingBottom: 5,
-    justifyContent: 'flex-start',
-    alignItems: 'flex-start',
-    backgroundColor: 'white',
-    borderRadius: 10,
-    overflow: 'hidden',
-    flex: 1
-  },
-  ListTitle: {
-    fontSize: 20,
-    fontWeight: 'bold',
-    borderBottomColor: '#0033A0',
-    borderBottomWidth: 2
-  },
-  ListTitleView: {
-    borderBottomColor: '#0033A0',
-    borderBottomWidth: 2
-  },
-  more: {
-    justifyContent: 'flex-end'
-  }
-})
-
 class Standings extends React.Component {
   constructor (props) {
     super(props)
 
     this.state = {
       allTeams: [],
+      allUsers: [],
+      allTeamsPPM: [], // PPM = points per member
       shownNumber: this.props.shownNumber | 3,
       topNumber: this.props.topNumber | 3,
-      isLoading: true
+      isLoading: true,
+      showScoresByTeam: true,
+      showPointsPerMember: false,
+      userTeamNum: undefined,
+      userID: undefined
     }
   }
 
   componentDidMount () {
+    // Create arrays for team list, one for team total scores and one for team scores per member.
     const teams = []
     this.props.firebase.getTeams().then(snapshot => {
       snapshot.forEach(doc => {
         teams.push({ id: doc.id, ...doc.data() })
       })
       // SortedTeams is an array that sorts the teams' points in descending order
       const sortedTeams = [].concat(teams).sort((a, b) => a.points < b.points)
+      const sortedTeamsPPM = [].concat(teams).sort((a, b) => (a.points / a.size) < (b.points / b.size))
       this.setState({ allTeams: sortedTeams, isLoading: false })
+      this.setState({ allTeamsPPM: sortedTeamsPPM, isLoading: false })
+    })
+
+    // Get list of users, for switching scoreboard to display people instead of teams.
+    const users = []
+    this.props.firebase.getUsers().then(snapshot => {
+      snapshot.forEach(doc => {
+        users.push({ id: doc.id, ...doc.data() })
+      })
+      const sortedUsers = [].concat(users).sort((a, b) => a.points < b.points)
+      this.setState({ allUsers: sortedUsers, isLoading: false })
+    })
+    
+    // Get the team number and user ID of the current user.
+    // These are for highlighting the user's position in the teams and users scoreboards
+    this.props.firebase.checkAuthUser(user => {
+      if(user){
+        this.props.firebase.getUserData(user.uid).then(data => {
+          this.setState({userTeamNumber: data.data().team})
+          this.setState({userID: data.data().uid})
+        })
+      }
     })
```

Comment:
You'll probably want to add an activity indicator while these are all loading. You can use `Promise.all()` and pass it an array of all your promises and that will only run once all of the promises have returned. [Here are the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all)